translators/postgres.go

```diff
@@ -23,19 +23,17 @@ func (p *Postgres) CreateTable(t fizz.Table) (string, error) {
 	for _, c := range t.Columns {
 		if c.Primary {
```

Comment:
Can you move the primary check after adding the column itself? That's the way it's done with PostgreSQL, so it should work the same way. :)

Reply:
I was trying to avoid having a second if c.Primary bellow but if it's ok for you when i'm doing that!